### DISCUSSION TITLE
How does `at()` work?

### DISCUSSION THREAD

**Discussion by Harold7665** [atoti/atoti#678] [Q&A]
Hello,

I am working on an unbalanced tree:
![Tree](https://user-images.githubusercontent.com/7359377/187941912-3f328d91-a80a-42e1-8428-ad48fb4236e7.png)

On the level 6, at the member X, I need to get the members CCC and DDD from another branch of the tree, at the level 3.
In order to do that, I have tried to use the 'at' function.
This way gives only 39,375 at CCC and DDD (I mean, only the current level 3).
```
tt.at(
    plan, 
    {
        l['Level 3']: 'CCC'
    }
)
```
This way gives me 39,375 at all the members at any level 3 of the tree.
```
tt.at(
    plan, 
    {
        l['Level 1']: 'AAA',
        l['Level 2']: 'BBB',
        l['Level 3']: 'CCC'
    }
)
```
Then I have to use this parent_value in order to get 39,375 at the member X of the level 6.
```
tt.parent_value(m, degrees={h['Account Tree']: 3})
```
I don't understand how 'at' is working. Since the first example, I was expecting to get 39,375 on all rows.

**Comment by tibdex** [comment]
I Harold, can you please clarify a bit what you want by providing a small but exhaustive example containing a hierarchy with at most 3 levels and two measures `A` and `B` where `A` have whatever values and `B` shows the values you want to get from `A` at different locations? Thanks.
